WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/ReaderViewPage.kt

```diff
@@ -8,53 +8,74 @@ import junit.framework.TestCase
 import org.wordpress.android.support.WPSupportUtils
 
 class ReaderViewPage {
-    var mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-    var mLikerContainerId = "org.wordpress.android.prealpha:id/liker_faces_container"
-    var mRelatedPostsId = "org.wordpress.android.prealpha:id/container_related_posts"
-    var mFooterId = "org.wordpress.android.prealpha:id/layout_post_detail_footer"
-    var mLikerContainer = mDevice.findObject(UiSelector().resourceId(mLikerContainerId))
-    var mRelatedPostsContainer = mDevice.findObject(UiSelector().resourceId(mRelatedPostsId))
-    var mSwipeForMore = mDevice.findObject(UiSelector().textContains("Swipe for more"))
-    var mFooter = mDevice.findObject(UiSelector().resourceId(mFooterId))
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    private val likerContainer = device.findObject(UiSelector().resourceId(buildResourceId("liker_faces_container")))
+    private val relatedPostsContainer = device.findObject(
+        UiSelector().resourceId(buildResourceId("container_related_posts"))
+    )
+    private val swipeForMore = device.findObject(UiSelector().textContains("Swipe for more"))
+    private val savePostsForLater = device.findObject(UiSelector().text("Save Posts for Later"))
+    private val ok = device.findObject(UiSelector().text("OK"))
```

Comment:
maybe we could rename this as `okButton` so that it's clearer what this `ok` is:

```suggestion
    private val okButton = device.findObject(UiSelector().text("OK"))
```

Reply:
2707d74350732c8f415ba68f7d5472b8b059ae08 Done!